### PR TITLE
Add create button and page

### PR DESCRIPTION
**Pull request by gracenng**
First PR in a while, let me know!

### CODE REVIEW

File: mobile/src/src/components/Create/CreatePage.jsx
Comment (trentstauff) [inline mobile/src/src/components/Create/CreatePage.jsx:5]: delete extra white space

File: mobile/src/src/components/Create/CreatePage.jsx
Comment (trentstauff) [inline mobile/src/src/components/Create/CreatePage.jsx:10]: delete line

File: mobile/src/src/components/Create/index.jsx
Comment (trentstauff) [inline mobile/src/src/components/Create/index.jsx:3]: add newline at end of file

File: mobile/src/src/components/Create/CreatePage.jsx
Comment (trentstauff) [inline mobile/src/src/components/Create/CreatePage.jsx:22]: newline end of file

File: mobile/src/src/navigators/TabNavigator.jsx
Comment (trentstauff) [inline mobile/src/src/navigators/TabNavigator.jsx:69]: Do you need this comment?

File: mobile/src/src/navigators/TabNavigator.jsx
Comment (trentstauff) [inline mobile/src/src/navigators/TabNavigator.jsx:60]: As you can see, ({ color}) isnt centered. Im going to stop the review now, install the prettier extension and just go through your files youve touched and do ctrl+S, and upload the changes

Comment (trentstauff) [reply mobile/src/src/navigators/TabNavigator.jsx:60]: Other than that- i havent tested the code locally yet, so i will do that and let you know. Code looks good though grace, good work.

**Review by trentstauff — CHANGES_REQUESTED** [review]
Just install prettier and clean code up- i also need to test it after so dont merge yet

**Comment by gracenng** [comment]
image
 tested this morning after re-cloning

**Review by trentstauff — APPROVED** [review]
Looks good, nice work Grace.

**Pull request merged**
